src/ansible/roles/ad/tasks/main.yml

```diff
@@ -84,32 +67,12 @@
   register: result
   failed_when: "result.rc != 255 and result.rc != 0"
   changed_when: "result.rc == 0"
+  until: "result.rc == 0"
```

Comment:
Please, add a comment why retry is needed.

Reply:
done